File: Assignment 2/Switch case/vowelconsonant.cpp
#include<iostream>
using namespace std;
class Vowel_Consonant
{
    char choice;
    public:void Case()
    {
        cout<<"Enter a character:";
        cin>>choice;
        switch(choice)
        {
            case 'a':cout<<"The entered character is a vowel";break;
            case 'e':cout<<"The entered character is a vowel";break;
            case 'i':cout<<"The entered character is a vowel";break;
            case 'o':cout<<"The entered character is a vowel";break;
            case 'u':cout<<"The entered character is a vowel";break;
            case 'A':cout<<"The entered character is a vowel";break;
            case 'E':cout<<"The entered character is a vowel";break;
            case 'I':cout<<"The entered character is a vowel";break;
            case 'O':cout<<"The entered character is a vowel";break;
            case 'U':cout<<"The entered character is a vowel";break;
            default:cout<<"The entered character is a consonant";
        }
    }
};
int main()
{
    Vowel_Consonant obj;
    obj.Case();
}

File: Java assignment(1-2)/java Assignment 2/Vowel_Consonant.java
import java.util.Scanner;
public class Vowel_Consonant 
{
public static void main(String args[])
{
System.out.println("Enter a character :");
Scanner sc = new Scanner(System.in);
char ch = sc.next().charAt(0);
if(ch == 'a'|| ch == 'e'|| ch == 'i' ||ch == 'o' ||ch == 'u'||ch == 'A'||ch=='E'||ch=='I'||ch=='O'||ch=='U')
{
 System.out.println("Given character is an vowel");
      }
else
{
         System.out.println("Given character is a consonant");
      }
 }
}